unitary/examples/quantum_checkers/players.py

```diff
@@ -0,0 +1,48 @@
+from random import randint
+import traceback
+
+
+class bot:
+    def __init__(self) -> None:
+        pass
+
+    def select_move(self, possible_moves):
+        pass
+
+
+class human_player(bot):
```

Comment:
HumanPlayer

See https://google.github.io/styleguide/pyguide.html#316-naming for naming guidelines.